fidesctl/src/fidesctl/cli/util_comands.py

```diff
@@ -23,6 +23,7 @@ def init(ctx: click.Context, fides_directory_location: str) -> None:
     """
 
     # Constants
+    separate = lambda: print("-" * 15)
```

Comment:
Not a fan, personally.

Reply:
this gives it a consistent look with apply/evaluate